### PR TITLE
Created an edit goal modal 

### PR DESCRIPTION
**Pull request by Draqode**
- [ ] Created a modal for editing created goals
- [ ] added a button for showing the goal editing modal

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "author": "Team Plato",
     "license": "MIT",
     "dependencies": {
+        "@reduxjs/toolkit": "^1.6.1",
```
Comment (Asylcreek) [inline package.json:19]: Please uninstall this package from the root directory and add it into the client's package.json

**Review by Asylcreek — COMMENTED** [review]
Please look at your EditGoal js files, especially the .styled.js file, and add some spaces between the exports. The spaces would improve readability

File: package.json (hunk 1 above)
Comment (Draqode) [reply package.json:19]: okay will do that now

**Comment by Draqode** [comment]
@abiona01 This is the pull request i wanna link to issue no 427

**Comment by Draqode** [comment]
@abiona01 fixes #427

**Comment by Draqode** [comment]
fixes #427

**Comment by abiona01** [comment]
Great

**Pull request merged**